backend/codesirius/problems/models/base_submission.py

```
@@ -0,0 +1,43 @@
+
+    problem = models.ForeignKey(
+        Problem,
+        on_delete=models.CASCADE,
+    )
+    code = models.TextField(max_length=10000)
+    language = models.ForeignKey(
+        Language,
+        on_delete=models.CASCADE,
+    )
+    verdict = models.CharField(
+        max_length=128, choices=Verdict.choices, default=Verdict.PENDING
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)